tutorials/MD/03_MD_Setup_Production_Run.ipynb

```diff
@@ -0,0 +1,274 @@
+    "\n",
+    "`% cd ../`\n",
+    "\n",
+    "`% mkdir run`\n",
+    "\n",
+    "`% cd run`\n",
+    "\n",
+    "The `grompp` command will read the information of the system that we will provide (coordinates, topologies and simulation parameters) and will generate a run input file:\n",
+    "\n",
+    "`% gmx grompp -c ../setup/system.gro -p ../setup/topol.top -f /home/shared/data/em.mdp -o em.tpr`\n",
+    "\n",
+    "We cannot run the simulation on the login nodes of ARC; these can only be used to prepare the system for the simulation. Therefore instead of typing the `mdrun` command -that will initiate the energy minimisation- directly on the command line, we will submit a script that will submit the job to the job scheduler. Copy this script to your working directory:\n",
+    "\n",
+    "`% cp /home/shared/data/submit_em.sh .`\n",
+    "\n",
+    "You can explore its contents and see that the last line in the file contains the `mdrun` command by typing: \n",
+    "\n",
+    "`% cat submit_em.sh`\n",
+    "\n",
+    "The file contents will be printed on your terminal.\\\n",
+    "Now submit it to the cluster queue.\n",
+    "\n",
+    "`% sbatch submit_em.sh`\n",
+    "\n",
+    "It will take a few minutes to run, depending on the waiting times of the queue. Check on the status of the run by typing:\n",
+    "\n",
+    "`% squeue -u username`\n",
+    "\n",
+    "Remember to replace `username` with your own username! It should print something like this:\n",
+    "\n",
+    "`             JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON) \n",
+    "           1311337       htc       EM bioc1550 PD       0:00      1 (Priority) \n",
+    "`\n",
+    "\n",
+    "The `PD` in the fifth column denotes that the job is in the queue and has not started running yet. It will change to `R` once it starts running:\n",
+    "\n",
+    "`             JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON) \n",
+    "           1311337       htc       EM bioc1550  R       1:26      1 arcus-htc-node110 \n",
+    "`\n",
+    "\n",
+    "If the job has finished (or if it has failed), the above command will print nothing:\n",
+    "\n",
+    "`JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON)`\n",
+    "\n",
+    "If you made a mistake and need to cancel this job, type:\n",
+    "\n",
+    "`% scancel JOBID`\n",
+    "\n",
+    "`JOBID` should be replaced by the ID of this particular job (printed in the first column; see above).\n",
+    "\n",
+    "While the job is running, you can monitor its progress by typing: \n",
+    "\n",
+    "`% tail -n 12 em.log`\n",
+    "\n",
+    "This will show you what step the simulation is at or if it has finished.\\\n",
```

Comment:
Can we make them look for the gromacs quote as an indicator that the job has finished properly?

Reply:
It is not printed at the end of the .log file